roles/transformers/vars/main.yml

```diff
@@ -4,10 +4,14 @@ temp_dir: /tmp/ansible_transformation
 
 imagemagick_home: "/usr/lib64/ImageMagick-{{ imagemagick_version[0:6] }}"
 imagemagick_rpm: "https://nexus.alfresco.com/nexus/service/local/repositories/thirdparty/content/org/imagemagick/imagemagick-distribution/{{ imagemagick_version }}/imagemagick-distribution-{{ imagemagick_version }}-linux.rpm"
+imagemagick_sha1_checksum: "{{ lookup('url', 'https://nexus.alfresco.com/nexus/service/local/repositories/thirdparty/content/org/imagemagick/imagemagick-distribution/{{ imagemagick_version }}/imagemagick-distribution-{{ imagemagick_version }}-linux.rpm.sha1', username=lookup('env', 'NEXUS_USERNAME'), password=lookup('env', 'NEXUS_PASSWORD')) }}"
```

Comment:
I think this one is also public so we probably don't need the credentials here either.